guides/common/attributes-titles.adoc

```diff
@@ -50,9 +50,11 @@ ifdef::satellite[]
 :InstallingSmartProxyDocTitle: Installing Capsule Server
 :ManagingConfigurationsPuppetDocTitle: Managing Configurations Using Puppet Integration in {ProjectName}
 :PlanningDocTitle: {Project} Overview, Concepts, and Deployment Considerations
+:QuickstartDocTitle: Quickstart Guide
```

Comment:
I'd suggest a title without the word "guide", ideally using gerundium to stick with the new title style.

Reply:
So, just "Quickstart"?

Reply:
I think it's fine for now, we don't have our Content Strategist to discuss it at the mo.